mix.exs

```diff
@@ -47,7 +47,7 @@ defmodule ExAws.Mixfile do
   defp package do
     [
       description: description(),
-      files: ["priv", "lib", "config", "mix.exs", "README*"],
+      files: ["priv", "lib", "config", "mix.exs", "README*, LICENSE"],
```

Comment:
Should this be `["priv", "lib", "config", "mix.exs", "README*", "LICENSE"],` instead?

Reply:
ups, yeah

Reply:
should be fixed now